src/service/url-replacements-impl.js

```diff
@@ -947,6 +983,9 @@ export class UrlReplacements {
         } else {
           binding = binding.async || binding.sync;
         }
+        if (opt_contextDoc) {
+          args.unshift(opt_contextDoc);
+        }
```

Comment:
^^ I'll admit, I don't love this. But the big addition on line 400 needs access to the creative's document. I considered putting that in opt_bindings and replacing this if statement with "if (name=='HTML_ATTR') { args.unshift(opt_bindings['srcdoc'];}" but that seemed less elegant. I also considered just doing the ${htmlAttr(...)} solution directly in amp-analytics.js, since that's where we're having to plumb the document in from, but that seemed even less elegant. So...this seems like the best to me. Open to other suggestions.